pkg/client/request_test.go

```diff
@@ -242,7 +234,7 @@ func TestTransformResponse(t *testing.T) {
 		{Response: &http.Response{StatusCode: 200, Body: ioutil.NopCloser(bytes.NewReader(invalid))}, Data: invalid},
 	}
 	for i, test := range testCases {
-		r := NewRequest(nil, "", uri, testapi.Codec(), true, true)
+		r := NewRequest(nil, "", uri, "v1beta1", testapi.Codec(), true, true)
```

Comment:
Examples to follow:
https://github.com/GoogleCloudPlatform/kubernetes/blob/3f28badae31b935aae72d33d63220923e2cac162/pkg/controller/replication_controller_test.go#L39